experimenter/tests/integration/nimbus/test_demo_app_integration.py

```diff
@@ -7,31 +9,46 @@
 from nimbus.pages.experimenter.summary import SummaryPage
 
 
-def navigate_to_demo_app_frontend(selenium):
-    demo_app_url = "http://demo-app-frontend:3000/"
-    selenium.get(demo_app_url)
+class DemoAppPage:
+    def __init__(self, driver):
+        self.driver = driver
 
+    def navigate_to(self):
+        demo_app_url = "http://demo-app-frontend:3000/"
+        self.driver.get(demo_app_url)
```

Comment:
This class is great, but please model it after our other classes found [here](https://github.com/mozilla/experimenter/tree/main/experimenter/tests/integration/nimbus/pages/experimenter). Since we will keep this demo app around for our testing, it will be good to have a page model for it that we can update as we add features.

If you follow the style that the other classes have, you'll have access to a lot of the built in methods we've implemented and it should make things a lot easier.